src/main/java/com/pyranid/KotlinDefaultResultSetMapper.kt

```diff
@@ -1,12 +1,18 @@
 package com.pyranid
 
+import org.postgresql.util.PGobject
```

Comment:
If `import` is removed and absolute references are used in code, we can "dynamically" access Postgres at runtime instead of having an import fail for users that do not use Postgres.  For example -

```
    } else if ("org.postgresql.util.PGobject".equals(resultSetValue.getClass().getName())) {
      org.postgresql.util.PGobject pgObject = (org.postgresql.util.PGobject) resultSetValue;
      return pgObject.getValue();
    }
```